modules/persistent-storage-csi-cloning-using.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-cloning.adoc
+
+[id="persistent-storage-csi-cloning-using_{context}"]
+= Using a cloned PVC as a storage volume
+
+A newly cloned PVC can be consumed, cloned, snapshotted, or deleted independently of its original dataSource PVC.
+
+Pods can access storage by using the cloned PVC as a volume. For example:
+
+.Use CSI volume clone in the Pod
+[source,yaml]
+----
+kind: Pod
+apiVersion: v1
+metadata:
+  name: mypod
+dataSource:
+  name: new-src-pvc <1>
+volumes:
+  - name: pvcclone
+    persistentVolumeClaim:
+      claimName: pvc-1-clone <2>
```

Comment:
Replacing what I had with your example. Keeping `dataSource` and annotation in example but moving to `spec`. If this is not helpful or accurate, let me know and I'll remove.

Reply:
> Copied from:
> https://docs.google.com/document/d/12fF8Xeg0K7v3hVYQCjthWr8_yN5hkvLFONueNrP7wIQ/edit
> 
> ```
> kind: Pod
> apiVersion: v1
> metadata:
>   name: mypod
> spec:
>   containers:
>     - name: myfrontend
>       image: dockerfile/nginx
>       volumeMounts:
>       - mountPath: "/var/www/html" 
>         name: mypd 
>   volumes:
>     - name: mypd
>       persistentVolumeClaim:
>         claimName: pvc-1-clone 
> ```
> 
> You can see it's exactly the same pod as in the linked example, only the clam name on the last line is now `pvc-1-clone` to get data from the cloned volume instead of the original.

Replacing what I had with your example. Keeping `dataSource` and annotation in example but moving to `spec`. If this is not helpful or accurate, let me know and I'll remove.